src/pages/api/auth/[...nextauth].ts

```diff
@@ -91,7 +91,12 @@ export default NextAuth({
 					token.refresh_token = refresh_token
 				} catch (err) {
 					console.error('failed to refresh token set', err)
-					return { ...token, error: AuthErrors.RefreshAccessTokenError }
+					return {
+						...token,
+						// NOTE: temporary prefix of [next-auth] for existing log monitor
+						// TODO: abstract logging so everything is grouped
+						error: `[next-auth] ${AuthErrors.RefreshAccessTokenError}`,
```

Comment:
We should update this check as well:
https://github.com/hashicorp/dev-portal/blob/13562d4a7220b94c01a3cfdf58767519f8e9ea8d/src/hooks/use-authentication/index.ts#L46-L47